java/src/main/java/org/rocksdb/RocksDB.java

```diff
@@ -307,9 +309,12 @@ public static RocksDB open(final DBOptions options, final String path,
     db.storeOptionsInstance(options);
 
     for (int i = 1; i < handles.length; i++) {
-      columnFamilyHandles.add(new ColumnFamilyHandle(db, handles[i]));
+      final ColumnFamilyHandle columnFamilyHandle = new ColumnFamilyHandle(db, handles[i]);
+      db.ownedColumnFamilyHandles.add(columnFamilyHandle);
```

Comment:
This feels a little strange. I think it would be better to switch these, e.g.:

```java
for (int i = 1; i < handles.length; i++) {
      columnFamilyHandles.add(new ColumnFamilyHandle(db, handles[i]));	      final ColumnFamilyHandle columnFamilyHandle = new ColumnFamilyHandle(db, handles[i]);
      columnFamilyHandles.add(columnFamilyHandle);
}
 db.ownedColumnFamilyHandles.addAll(columnFamilyHandles);
```

Please apply to others as well.